scripts/update/jellyfin.sh

```diff
@@ -68,15 +62,79 @@ if [[ -f /install/.jellyfin.lock ]]; then
     fi
     #
     if ! check_installed jellyfin; then
-        echo_info "Moving Jellyfin to apt-managed installation"
+        echo_info "Updating Jellyfin installation using apt."
         #
-        # Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
-        curl -s https://repo.jellyfin.org/$DIST_ID/jellyfin_team.gpg.key | gpg --dearmor > /usr/share/keyrings/jellyfin-archive-keyring.gpg 2>> "${log}"
-        echo "deb [signed-by=/usr/share/keyrings/jellyfin-archive-keyring.gpg arch=$(dpkg --print-architecture)] https://repo.jellyfin.org/$DIST_ID $DIST_CODENAME main" > /etc/apt/sources.list.d/jellyfin.list
+        # Make sure universe is enabled so that ffmpeg can be satisfied.
+        sudo add-apt-repository universe
+
+        # Handle some known alternative base OS values with 1-to-1 mappings
+        # Use the result as the repository base OS
+        ARCHITECTURE="$(dpkg --print-architecture)"
+        BASE_OS="$(awk -F'=' '/^ID=/{ print $NF }' /etc/os-release)"
+        case "${BASE_OS}" in
```

Comment:
replicated code should be a function; however, this is useless code beyond the setting of the distro codename (as above)

Reply:
I will use _os_codename in both cases if you can confirm that raspbian is set to debian. If not, I will create a function.